src/screens/LoginPage/LoginPage.tsx

```diff
@@ -57,6 +64,53 @@ function loginPage(): JSX.Element {
   const [showPassword, setShowPassword] = useState<boolean>(false);
   const [showConfirmPassword, setShowConfirmPassword] =
     useState<boolean>(false);
+  const [showAlert, setShowAlert] = useState<PasswordValidation>({
+    lowercaseChar: true,
+    uppercaseChar: true,
+    numericValue: true,
+    specialChar: true,
+  });
+
+  const passwordValidationRegExp = {
+    lowercaseCharRegExp: new RegExp('[a-z]'),
+    uppercasaeCharRegExp: new RegExp('[A-Z]'),
+    numericalValueRegExp: new RegExp('\\d'),
+    specialCharRegExp: new RegExp('[!@#$%^&*()_+{}\\[\\]:;<>,.?~\\\\/-]'),
+  };
+  const handleLowercasePassCheck = (pass: string): void => {
+    const success = passwordValidationRegExp.lowercaseCharRegExp.test(pass);
+    if (success) {
+      setShowAlert((prevAlert) => ({ ...prevAlert, lowercaseChar: false }));
+    } else {
+      setShowAlert((prevAlert) => ({ ...prevAlert, lowercaseChar: true }));
+    }
+  };
+
+  const handleUppercasePassCheck = (pass: string): void => {
+    const success = passwordValidationRegExp.uppercasaeCharRegExp.test(pass);
+    if (success) {
+      setShowAlert((prevAlert) => ({ ...prevAlert, uppercaseChar: false }));
+    } else {
+      setShowAlert((prevAlert) => ({ ...prevAlert, uppercaseChar: true }));
+    }
+  };
+  const handleNumericalValuePassCheck = (pass: string): void => {
```

Comment:
@MahendraDani refactor all related methods like this

```ts
  const handleNumericalValuePassCheck = (pass: string): void => {
    if (passwordValidationRegExp.numericalValueRegExp.test(pass)) {
      setShowAlert((prevAlert) => ({ ...prevAlert, numericValue: false }));
    } else {
      setShowAlert((prevAlert) => ({ ...prevAlert, numericValue: true }));
    }
  };
```

Reply:
Or even better like this,

@MahendraDani refactor all related methods like this

```ts
  const handleNumericalValuePassCheck = (pass: string): void => {
      setShowAlert((prevAlert) => ({ 
          ...prevAlert, 
          numericValue: passwordValidationRegExp.numericalValueRegExp.test(pass)
      }));
  };
```

Reply:
Working on it

Reply:
@rishav-jha-mech 
Completed this via https://github.com/PalisadoesFoundation/talawa-admin/pull/1376/commits/7e1dea4c257ec1a903d4b9a6e04d0f48b76e490c

Reply:
> Or even better like this,
> 
> ```ts
>   const handleNumericalValuePassCheck = (pass: string): void => {
>       setShowAlert((prevAlert) => ({ 
>           ...prevAlert, 
>           numericValue: passwordValidationRegExp.numericalValueRegExp.test(pass)
>       }));
>   };
> ```

Yes I think this should be better. Doing this now

Reply:
> Or even better like this,
> 
> ```ts
>   const handleNumericalValuePassCheck = (pass: string): void => {
>       setShowAlert((prevAlert) => ({ 
>           ...prevAlert, 
>           numericValue: passwordValidationRegExp.numericalValueRegExp.test(pass)
>       }));
>   };
> ```

@rishav-jha-mech 
I tried this but it is not working as expected

Reply:
@rishav-jha-mech 
Please check the following screen recording

https://github.com/PalisadoesFoundation/talawa-admin/assets/116940083/5219b562-54bb-450c-81bc-1615e0773581

Reply:
Try once by adding  `!` mark in before the validation

Reply:
> Try once by adding `!` mark in before the validation

@rishav-jha-mech It is not working either

Reply:
@rishav-jha-mech 
Should we just go with the following code?
```typescript
  const handleNumericalValuePassCheck = (pass: string): void => {
      setShowAlert((prevAlert) => ({ 
          ...prevAlert, 
          numericValue: passwordValidationRegExp.numericalValueRegExp.test(pass)
      }));
  };
```

Reply:
@MahendraDani use this then if its working

```ts
  const handleNumericalValuePassCheck = (pass: string): void => {
    if (passwordValidationRegExp.numericalValueRegExp.test(pass)) {
      setShowAlert((prevAlert) => ({ ...prevAlert, numericValue: false }));
    } else {
      setShowAlert((prevAlert) => ({ ...prevAlert, numericValue: true }));
    }
  };
```

Reply:
> @MahendraDani use this then if its working
> 
> ```ts
>   const handleNumericalValuePassCheck = (pass: string): void => {
>     if (passwordValidationRegExp.numericalValueRegExp.test(pass)) {
>       setShowAlert((prevAlert) => ({ ...prevAlert, numericValue: false }));
>     } else {
>       setShowAlert((prevAlert) => ({ ...prevAlert, numericValue: true }));
>     }
>   };
> ```

@rishav-jha-mech 
This is already implemented via https://github.com/PalisadoesFoundation/talawa-admin/commit/7e1dea4c257ec1a903d4b9a6e04d0f48b76e490c